src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -1818,6 +2205,15 @@ - (instancetype)initWithPath:(const ConcreteClusterPath &)path
     return self;
 }
 
+- (instancetype)initWithEndpointID:(NSNumber * _Nullable)endpointID clusterID:(NSNumber * _Nullable)clusterID
+{
+    if (self = [super init]) {
+        _endpoint = endpointID;
```

Comment:
This is broken.  The attribute backed by this ivar is claimed to be non-nullable, but this might be assigning nil to the ivar.

As the comments on these interfaces say, they represent concrete paths, without wildcards.  If you need to represent possibly-wildcard paths, you need a different set of interfaces.  You can't use MTRAttributePath/MTREventPath to represent things with wildcards.

Reply:
Instead of nil, I will modify it to handle wildcard values by defining them. (It will be corrected at the end of the entire conversation.)
e.g.,`NSNumber * const MTRWildCardAttributeId = kInvalidAttributeId;`

Reply:
Please do not use `kInvalidAttributeId` in the API.  The fact that such a thing exists should really not be exposed.  The right representation for wildcards is nil.  You just need a path representation that can represent wildcard paths, instead of repurposing the one we have right now which is explicitly for concrete paths only.

Reply:
I understand your comment that you should create an interface separate from MTRAttributePath / MTREventPath. By the way, is there a reason you have to make it separately? Of course, I agree that the current implementation has a problem that wildcards are not processed, but since the properties in MTRAttributePath are the same, I think it's okay to use them together. Also, it is difficult to create another interface name because the meaning of the MTRAttributePath name is not the meaning of the output. Can you recommend another interface name? (Attribute, Event)

Reply:
> By the way, is there a reason you have to make it separately?

Well... we can't change `MTRAttributePath`, right?  What other options do you see?

>  Can you recommend another interface name?

MTRAttributeRequestPath, MTREventRequestPath?

Reply:
Thank you about your advice. I modified.